pkg/provider/doppler/client.go

```diff
@@ -37,6 +37,8 @@ const (
 	verifyTLSOverrideEnvVar                            = "DOPPLER_VERIFY_TLS"
 	errGetSecret                                       = "could not get secret %s: %s"
 	errGetSecrets                                      = "could not get secrets %s"
+	errDeleteSecrets                                   = "could not delete secrets %s: %s"
+	errPushSecrets                                     = "could not push secrets %s: %s"
```

Comment:
```suggestion
	errDeleteSecrets                                   = "could not delete secrets %s: %w"
	errPushSecrets                                     = "could not push secrets %s: %w"
```

Reply:
🙏 I went ahead and replaced the `%s` with `%w` in two adjacent lines as well.

Reply:
Thanks!